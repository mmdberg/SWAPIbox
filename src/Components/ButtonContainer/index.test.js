import React from 'react';
import { shallow } from 'enzyme';
import { ButtonContainer } from './index';

describe('Button Container', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<ButtonContainer />)
    expect(wrapper).toMatchSnapshot()
  })

  //test that onclick has been called with right parameters

})