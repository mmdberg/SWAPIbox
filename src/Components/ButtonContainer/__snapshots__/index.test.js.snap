// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Container should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="button"
          onClick={[Function]}
          to="/home/people/"
>
          People
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="button"
          onClick={[Function]}
          to="/home/planets/"
>
          Planets
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="button"
          onClick={[Function]}
          to="/home/vehicles/"
>
          Vehicles
</NavLink>,
      ],
      "className": "button-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "People",
          "className": "button",
          "onClick": [Function],
          "to": "/home/people/",
        },
        "ref": null,
        "rendered": "People",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Planets",
          "className": "button",
          "onClick": [Function],
          "to": "/home/planets/",
        },
        "ref": null,
        "rendered": "Planets",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Vehicles",
          "className": "button",
          "onClick": [Function],
          "to": "/home/vehicles/",
        },
        "ref": null,
        "rendered": "Vehicles",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="button"
            onClick={[Function]}
            to="/home/people/"
>
            People
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="button"
            onClick={[Function]}
            to="/home/planets/"
>
            Planets
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="button"
            onClick={[Function]}
            to="/home/vehicles/"
>
            Vehicles
</NavLink>,
        ],
        "className": "button-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "People",
            "className": "button",
            "onClick": [Function],
            "to": "/home/people/",
          },
          "ref": null,
          "rendered": "People",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Planets",
            "className": "button",
            "onClick": [Function],
            "to": "/home/planets/",
          },
          "ref": null,
          "rendered": "Planets",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Vehicles",
            "className": "button",
            "onClick": [Function],
            "to": "/home/vehicles/",
          },
          "ref": null,
          "rendered": "Vehicles",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
